docs/settings.md

```diff
@@ -66,6 +66,7 @@ Option legend:
 | `turn_servers`                       | array[]                                                          | `[]`       | 🖌️💻  | List of TURN servers, should be configured via the web interface or the OCC commands                                                                                              |
 | `signaling_servers`                  | array[]                                                          | `[]`       | 🖌️💻  | List of signaling servers, should be configured via the web interface or the OCC commands                                                                                         |
 | `signaling_mode`                     | string<br>`internal` or `external` or `conversation_cluster`     | `internal` |        | `internal` when no HPB is configured, `external` when configured, `conversation_cluster` is an experimental flag that is deprecated                                               |
+| `recording_servers`                  | array[]                                                          | `[]`       | 🖌️    | List of recording servers, should be configured via the web interface                                                                                                             |
```

Comment:
Can we move this below `turn_servers` so the sorting by type persists?

Reply:
Done.